src/main/java/com/team766/controllers/CanSparkMaxSmartMotionPIDController.java

```diff
@@ -117,42 +117,38 @@ public void setAntigravConstant(double k){
 	}
 
 	private void antigrav(){
-		mc1.set(antiGravK * Math.sin(mc1.getSensorPosition()));
+		mc.set(antiGravK * Math.sin(mc.getSensorPosition()));
```

Comment:
the `Math.sin` in this equation means that this is specialized for rotational joints (i.e. arms). if we wanted to use this to control an elevator, this would be a fixed power, independent of mechanism position.

if we don't want to generalize this, we should probably clarify this in the name of the class. `CanSparkMaxArmController` or something